products/windowsServer.md

```diff
@@ -50,6 +50,13 @@ releases:
     latest: 10.0.18363
     link: https://learn.microsoft.com/lifecycle/announcements/windows-server-1909-end-of-servicing
 
+-   releaseCycle: "1903"
+    releaseDate: 2019-05-21
+    eoas: 2020-12-08
+    eol: 2020-12-08
+    latest: 10.0.18362
+    link: https://learn.microsoft.com/en-us/lifecycle/products/windows-server-semiannual-channel
+
```

Comment:
Why have you moved this releaseCycle? Ordered by releaseDate this cycle needs to be at the old position.

Reply:
The release date was wrong. It said May 2018 instead of May 2019. So I fixed it and moved accordingly.

https://www.microsoft.com/en-us/windows-server/blog/2019/05/21/windows-server-version-1903-now-generally-available/

Reply:
Good catch :+1: